atest/robot/test_libraries/logging_with_logging.robot

```diff
@@ -16,18 +16,18 @@ Log with default levels
     Check log message  ${tc.kws[0].msgs[0]}  debug message     DEBUG
     Check log message  ${tc.kws[0].msgs[1]}  info message      INFO
     Check log message  ${tc.kws[0].msgs[2]}  warning message   WARN
-    Check log message  ${tc.kws[0].msgs[3]}  error message     WARN
-    Check log message  ${tc.kws[0].msgs[4]}  critical message  WARN
+    Check log message  ${tc.kws[0].msgs[3]}  error message     ERROR
+    Check log message  ${tc.kws[0].msgs[4]}  critical message  ERROR
     Check log message  ${ERRORS.msgs[0]}     warning message   WARN
-    Check log message  ${ERRORS.msgs[1]}     error message     WARN
-    Check log message  ${ERRORS.msgs[2]}     critical message  WARN
+    Check log message  ${ERRORS.msgs[1]}     error message     ERROR
+    Check log message  ${ERRORS.msgs[2]}     critical message  ERROR
 
 Log with custom levels
     ${tc} =  Check test case  ${TEST NAME}
     Check log message  ${tc.kws[0].msgs[1]}  below debug               TRACE
     Check log message  ${tc.kws[0].msgs[2]}  between debug and info    DEBUG
     Check log message  ${tc.kws[0].msgs[3]}  between info and warning  INFO
-    Check log message  ${tc.kws[0].msgs[4]}  above warning             WARN
+    Check log message  ${tc.kws[0].msgs[4]}  above warning             ERROR
```

Comment:
It would be good to test also a custom level between warning and error. That should be mapped to WARN because levels between info and warning are mapped to INFO. Anything above error should be ERROR.